cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/data/MapTotalsRepository.java

```diff
@@ -0,0 +1,14 @@
+package edu.asu.diging.cord19.explorer.core.data;
+
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+import edu.asu.diging.cord19.explorer.core.model.impl.MapTotalsImpl;
+
+@Repository
+public interface MapTotalsRepository extends PagingAndSortingRepository<MapTotalsImpl, String> {
```

Comment:
The second parameter of `PagingAndSortingRepository<MapTotalsImpl, String>` should be the type of the id property which is `Long` not `String` if I'm not mistaken?